src/mod.test.ts

```diff
@@ -363,15 +363,14 @@ if (typeof WebAssembly === 'undefined') {
     });
 
     test('test writes that span multiple blocks (w/small buffer)', async () => {
-      const res = await fetch('http://localhost:8124/src/mod.test.ts');
-      const result = await res.text();
+      const value = '9:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ'.repeat(18428 / 34);
```

Comment:
Fix a silly test stability issue: this test used to pass the content of the test file around. Anytime the test file changed, so did the length of the buffer. The unit test in question failed when transferring 18,428 bytes, so we pin it there. Additionally, we use a 34-character repeating readable ASCII sequence to make it easier to spot "missing" runs of characters.